docs/Antora/modules/howto_guides/pages/howto_connect_to_device.adoc

```diff
@@ -53,6 +53,25 @@ for device_info in instance.available_devices:
     if device_info.connection_string.startswith('daq.opcua://'):
         print('Name:', device_info.name, 'Address:', device_info.connection_string)
 ----
+C#::
++
+[source,csharp]
+----
+// See https://aka.ms/new-console-template for more information
```

Comment:
I think these comments can be removed (new-console-template)

Reply:
done